src/components/PageContainer/style.ts

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components/native";
+import { ColorScheme, FontScheme } from "../../styles/globalStyles";
+
+export const Container = styled.ScrollView`
+  display: flex;
+  flex: 1;
+  background-color: ${ColorScheme.background.primary};
+  padding: ${FontScheme.size.heading}px;
+  flex-grow: 1;
+`;
```

Comment:
Se for usar o SafeAreView (que acho uma boa idéia), você irá precisar ajustar o padding top, já que o SafeAreaView já traz um espaçamento no topo. Então o padding do container do PageContainer ficaria algo do tipo:
`padding: ${FontScheme.size.small}px ${FontScheme.size.heading}px ${FontScheme.size.heading}px ${FontScheme.size.heading}px;`
Que equivale a: padding-top, padding-right, padding-bottom e padding-left.

Outra coisa que irá precisar fazer é remover o margin-top do Header da página de SignUp.

Reply:
Atualizei o Container para usar SafeAreaView e configurei o padding conforme sugerido, usando padding-top, padding-right, padding-bottom e padding-left com valores de FontScheme